test/common/benchmark.js

```diff
@@ -20,10 +20,27 @@ function runBenchmark(name, args, env) {
 
   const mergedEnv = Object.assign({}, process.env, env);
 
-  const child = fork(runjs, argv, { env: mergedEnv });
+  const child = fork(runjs, argv, { env: mergedEnv, stdio: 'pipe' });
+
+  let stdout = '';
+  child.stdout.on('data', (line) => {
+    const thisLine = line.toString();
```

Comment:
Can we rather use `child.stdout.setEncoding('utf8')`? That way we don’t have to explicitly convert + we handle multi-byte characters properly, if that should ever be an issue